### DISCUSSION TITLE
Can I choose a specific workspace by passing argument to `nvim` in terminal?

### DISCUSSION THREAD

**Discussion by amalgame21** [nvim-neorg/neorg#1070] [Q&A]
For example, I have a workspace named "mynote" with path "~/mynote"
I have to go through 3 step to enter the workspace index:
1. type `nvim` in terminal
2. type `:Neorg workspace mynote`
3. type `:Neorg index`

I know I can make a keymap to combine 2 and 3,
but are that any other ways that I can even pass some argument to `nvim` so I am already in the specific workspace when it is launched?
I tried `nvim ~/mynote/index.norg`, but it does not put me in the `mynote` workspace...
Thanks!

**Comment by amalgame21** [comment]
I think the `default_workspace` option in `core.dirman` is what I am looking for!

**Comment by Anrock** [comment]
You can create shell alias for `nvim +Neorg workspace mynote +Neorg index`, that's what I did for my notes.
